app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -52,6 +52,7 @@ interface Pixel {
         WEB_RENDERER_GONE_KILLED("m_d_wrg_k"),
         BROKEN_SITE_REPORTED("m_bsr"),
         BROKEN_SITE_REPORT("epbf"),
+        WEB_VIEW_REDIRECT_HEADERS("m_wb_rh"),
```

Comment:
@cmonfortep do we still want this pixel or should I remove it?

Reply:
If we are moving away from `shouldOverrideUrlLoading`, this can be removed. 👍

Reply:
Nice, removed!